dash/dash.py

```diff
@@ -77,6 +77,22 @@
 from ._jupyter import jupyter_dash, JupyterDisplayMode
 from .types import RendererHooks
 
+dash_design_kit_version = None
+try:
+    import dash_design_kit
+
+    dash_design_kit_version = dash_design_kit.__version__
```

Comment:
Maybe use `importlib.metadata` instead, so it'll still work if DDK changes the name of the `__version__` property (and also we don't have to load DDK into memory)

```suggestion
    dash_design_kit_version = importlib.metadata.version("dash_design_kit")
```

Reply:
Done!

Reply:
Ended up being better anyway because then you don't have to import the package